### DISCUSSION TITLE
Can't see results [No data available for this test.]

### DISCUSSION THREAD

**Discussion by petar-dambovaliev** [TechEmpower/FrameworkBenchmarks#6518] [General]
I am running `./tfb --mode verify --test mybench`, copy the result json content to https://tfb-status.techempower.com/share
and i get `No data available for this test.`

**Comment by NateBrady23** [comment]
can you share the json file?

**Comment by joanhey** [comment]
You need to use `--mode benchmark`
https://github.com/TechEmpower/FrameworkBenchmarks/discussions/6505#discussioncomment-576626

**Reply by petar-dambovaliev** [reply]
Sorry, i don't remember what this issue was about.
I gave up on including my framework to these benchmarks because i don't view them as useful.
And this has nothing to do with the project but the other framework vendors.
Look at their benchmarks labelled as `realistic` and at the same time they have used custom specialized memory allocators and a million other tricks to squeeze every millisecond.
This is everything but realistic and it beats the purpose of the whole idea.
Nobody writes code like that.
